scripts/package.json

```diff
@@ -7,9 +7,13 @@
         "type-check": "tsc --build"
     },
     "dependencies": {
+        "chalk": "^5.0.0",
```

Comment:
If I understand well [chalk](https://github.com/chalk/chalk) is only for making more fancy colors in command line. 
Do we really need that if it is only for development?

Reply:
Yea we can live without colors and I agree we should be carefull about dependecies, but since this is just development dependency I think we don't have to care that about it too much. It costs you nothing since it's not bundled into final app anyway and if it will make your life and DX better easier then use it.

I plan to use it for all current/future utility scripts because colors help you to not miss something in output and DX it's just better.